doc/source/cluster/cluster_under_construction/getting-started.rst

```diff
@@ -26,13 +33,25 @@ One of Ray's strengths is the ability to leverage multiple machines for
 distributed execution. Ray can, of course, be run on a single machine (and is
 done so often), but the real power is using Ray on a cluster of machines.
 
-Ray can automatically interact with the cloud provider to request or release
-instances. You can specify :ref:`a configuration <cluster-config>` to launch
-clusters on :ref:`AWS, GCP, Azure (community-maintained), Aliyun (community-maintained), on-premise, or even on
-your custom node provider <cluster-cloud>`. Ray can also be run on :ref:`Kubernetes <kuberay-index>` infrastructure.
-Your cluster can have a fixed size
-or :ref:`automatically scale up and down<cluster-autoscaler>` depending on the
-demands of your application.
+A Ray Cluster can be created from virtual machines using the :ref:`Ray Cluster Launcher<ref-cluster-quick-start-vms-under-construction>`, which is officially supported on AWS and GCP and community supported on Azure and Aliyun, or created from machines managed by Kubernetes via :ref:`KubeRay<kuberay-index>`, which is the officially supported way to deploy Ray on Kubernetes.
+Ray can also be :ref:`deployed on-premise<cluster-private-setup-under-construction>` or even onto infrastructure not listed here by :ref:`providing a custom node provider<additional-cloud-providers-under-construction>`.
+
+Your cluster can have a fixed size, or :ref:`leverage autoscaling<cluster-autoscaler-under-construction>` to automatically request additional or release unused nodes depending on the demands of your application.
```

Comment:
Nit/suggestion: `to automatically request additional resources or release unused nodes, according to the demands of your application.`